code/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -183,7 +183,7 @@
     "    --STAR-index reference_data/STAR_Index/ \\\n",
     "    --gtf reference_data/Homo_sapiens.GRCh38.103.chr.reformatted.gene.ERCC.gtf \\\n",
     "    --container container/rna_quantification.sif \\\n",
-    "    --mem 40G"
+    "    --mem 40G -n"
```

Comment:
why `-n` here? please clean it up if that's not the intention

Reply:
cleaned